src/CMakeLists.txt

```diff
@@ -401,9 +401,14 @@ if(WITH_DPDK)
   list(APPEND ceph_common_deps common_async_dpdk)
 endif()
 
-if(WITH_BLUESTORE_PMEM)
+if(WITH_BLUESTORE_PMEM OR WITH_RBD_RWL)
   if(WITH_SYSTEM_PMDK)
-    find_package(pmem REQUIRED)
+    if(WITH_BLUESTORE_PMEM)
+      find_package(pmem REQUIRED)
+    endif()
+    if(WITH_RBD_RWL)
+      find_package(pmemobj REQUIRED)
```

Comment:
please note we don't have `Findpmemobj.cmake`. probably you could improve `Findpmem.cmake` to support the `COMPONENTS` option?

Reply:
Let me try.